onnx/backend/test/case/model/local-functions.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+from typing import Sequence
+
+from onnx import helper, parser, checker, compose, version_converter, \
+    ModelProto, GraphProto, ValueInfoProto, TensorProto, SparseTensorProto, \
+    FunctionProto, NodeProto
+
+
+class LocalFunctions(Base):
+
+    @staticmethod
+    def export():  # type: () -> None
+
+        ops = [
+            helper.make_opsetid("", 10),
+            helper.make_opsetid("local", 10)
+        ]
+
+        g = GraphProto()
```

Comment:
I suggest using onnx parser api for test model creation. It is more readable and easier to use.